src/cleaner/__init__.py

```diff
@@ -1,12 +1,19 @@
-import json, argparse, os, datetime as dt, xxhash
-from azure.cosmos import CosmosClient, exceptions
-from dotenv import load_dotenv
+import json
+import os
+import datetime as dt
+import xxhash
 
 class Cleaner:
 
     def __init__(self, county):
         self.county = county.lower()
 
+    def add_parsing_date(self, input_dict: dict, out_file: dict) -> dict:
+        # This will add the date of parsing to the final cleaned json file
+        today_date = dt.datetime.today().strftime('%Y-%m-%d')
+        out_file['parsing_date'] = today_date
+        return out_file
```

Comment:
Ok this is totally a nitpick, not a thing you need to change. I would make a choice (and either is usually fine) about whether methods like this alter their inputs (`out_file['parsing_date'] = today_date`) or return a new object that is altered (`return {'parsing_date': today_date, ...out_file}`) without modifying the input version.
If you alter the input, I wouldn't return anything, to make it clear that that's what's happening vs the other method. All of this really only makes a difference in big codebases, so feel free to ignore it :)